src/backend/core/tests/test_api_users.py

```diff
@@ -17,8 +16,11 @@ def test_api_users_list_anonymous():
     factories.UserFactory()
     client = APIClient()
     response = client.get("/api/v1.0/users/")
-    assert response.status_code == 404
-    assert "Not Found" in response.content.decode("utf-8")
+    assert response.status_code == 401
+    print(response.json())
+    assert "Authentication credentials were not provided." in response.content.decode(
+        "utf-8"
+    )
 
 
 def test_api_users_list_authenticated():
```

Comment:
This test is now broken because we succesfully retrieve a list (200 != 404). I feel that listing all users should be allowed, as query searches can list all users anyway. Do you confirm, @sampaccoud ?